modules/tinymce/src/core/main/ts/init/Render.ts

```diff
@@ -45,22 +45,15 @@ const loadLanguage = (scriptLoader: ScriptLoader, editor: Editor) => {
   }
 };
 
-const loadTheme = (scriptLoader: ScriptLoader, editor: Editor, suffix: string): Promise<void> => {
+const loadTheme = (scriptLoader: ScriptLoader, editor: Editor, suffix: string): void => {
```

Comment:
deepscan notes `scriptLoader` is unused

Reply:
Yeah had just seen that myself as well. Have removed the unused argument.